Orange/widgets/evaluate/owparameterfitter.py

```diff
@@ -252,6 +253,54 @@ def __get_index_at(self, point: QPointF) -> Optional[int]:
         return None
 
 
+class RangePreview(QWidget):
+    def __init__(self):
+        super().__init__()
+        font = self.font()
+        font.setPointSize(font.pointSize() - 3)
+        self.setFont(font)
+
+        self.__steps: Optional[Sequence[int]] = None
+        self.setSizePolicy(QSizePolicy.MinimumExpanding, QSizePolicy.Preferred)
+
+    def minimumSizeHint(self):
+        return QSize(1, 20)
+
+    def set_steps(self, steps: Optional[Sequence[int]]):
+        self.__steps = steps
+        self.update()
+
+    def steps(self):
+        return self.__steps
+
+    def paintEvent(self, event):
+        if not self.__steps:
+            return
+        painter = QPainter(self)
+        metrics = QFontMetrics(self.font())
+        style = self.style()
+        rect = self.rect()
+
+        # Indent by the width of the radio button indicator
+        rect.adjust(style.pixelMetric(style.PM_IndicatorWidth)
+                    + style.pixelMetric(style.PM_CheckBoxLabelSpacing), 0, 0, 0)
+
+        last_text = f", {self.__steps[-1]}"
```

Comment:
![image](https://github.com/user-attachments/assets/39eda622-9ea1-4f85-ade2-48589bfa23c5)
I'd remove the `,` for a single value.